docs/contributor-guide.rst

```diff
@@ -135,3 +135,22 @@ Editor config
 
 - `python-black <https://atom.io/packages/python-black>`_
 - `linter-flake8 <https://atom.io/packages/linter-flake8>`_
+
+Overview of Resource and Collection interaction
+-----------------------------------------------
+  .. image:: https://github.com/Datatamer/unify-client-python/blob/charlottemoremen-contributorguide/docs/resource:collectionRoute.png?sanitize=true
+
+  .. image:: https://github.com/Datatamer/unify-client-python/blob/charlottemoremen-contributorguide/docs/resource:collectionRequest.png?sanitize=true
```

Comment:
We can't do absolute URLs here; since the image files are stored adjacent to the doc, it should work to use relative URLs (just the image file name).